pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -97,13 +127,22 @@ By convention, `full` indicates that the permission supports all other action id
 
 ## <a name="secrets"></a>Secret Store Permissions
 
-|                                                                                                                                 Permission string                                                                                                                                 | full | C | R | U | D |
+These permissions control access to the [Secrets API](/1.10/security/ent/secrets/secrets-api/). A Mesos framework must have
+permission granted to its DC/OS service account in order to access a given secret. If you are looking for information on how to launch
+Marathon applications using secrets see [Configuring services and pods to use secrets](/1.10/security/ent/secrets/use-secrets/).
+
+|                                                                                                                                 Resource identifier                                                                                                                                 | full | C | R | U | D |
 |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------|---|---|---|---|
 | `dcos:secrets:default:[<path-name>/]<secret-name>`<br> Controls access to individual [secrets](/1.10/security/ent/secrets/).                                                                                                                                                           | x    | x | x | x | x |
 | `dcos:secrets:list:default:/[<path>]`<br> Controls view access to the names of [secrets](/1.10/security/ent/secrets/).                                                                                                                                                                 |      |   | x |   |   |
 
 ## <a name="superuser"></a>Superuser Permissions
 
-|                                                                                                                                 Permission string                                                                                                                                 | full | C | R | U | D |
+Similar to the Windows `Administrator` or Linux `root` accounts, DC/OS has the
+concept of the `superuser`. A user with permission to perform any action on the `dcos:superuser` resource has complete, unrestricted access to any operation
```

Comment:
In Slack, our friend @gpaul said:

> in that sense, it [`dcos:superuser`] is not a resource in itself, but a meta resource

Please confirm that "the `dcos:superuser` resource" is what you mean by this.

Reply:
I'm going to stick with 'resource' for now so we don't introduce a new concept (meta-resource) in this document. In future we may have 'security policy' as a concept that more naturally describes  `dcos:superuser`.